core/config/project_settings.cpp

```diff
@@ -197,7 +197,7 @@ String ProjectSettings::localize_path(const String &p_path) const {
 			return "res://" + path;
 		}
 
-		String parent = path.substr(0, sep);
+		String parent = path.left(sep);
```

Comment:
`sep` is the result of `rfind()`, but the -1 case's check is already returned before this point. So it's safe.